recipes/openldap/all/conanfile.py

```diff
@@ -49,55 +49,38 @@ def validate(self):
             raise ConanInvalidConfiguration(
                 f"{self.name} is only supported on Linux")
```

Comment:
Better to print the same thing as you need for cli or profile for the reference 

```suggestion
                f"{self.ref} is only supported on Linux")
```